analytics_data_api/v0/serializers.py

```diff
@@ -241,3 +241,31 @@ class Meta(object):
         model = models.CourseActivityWeekly
         # TODO: Add 'posted_forum' here to restore forum data
         fields = ('interval_start', 'interval_end', 'course_id', 'any', 'attempted_problem', 'played_video', 'created')
+
+
+class VideoSerializer(ModelSerializerWithCreatedField):
+    # pipeline_video_id is renamed to video_id
+    video_id = serializers.CharField(required=True)
+
+    class Meta(object):
+        model = models.Video
+        fields = (
+            'video_id',
+            'encoded_module_id',
+            'duration',
+            'segment_length',
+            'start_views',
+            'end_views',
+            'created'
+        )
+
+
+class VideoTimelineSerializer(ModelSerializerWithCreatedField):
+    class Meta(object):
+        model = models.VideoTimeline
+        fields = (
+            'segment',
```

Comment:
it just occurred to me that in the data I generated "segment" is actually "seconds"... so instead of seeing 0,1,2,3... you will see 0,5,10,15...

We probably want the former?